zilliqa/src/api/erigon.rs

```diff
@@ -18,7 +18,7 @@ fn get_header_by_number(params: Params, node: &Arc<Mutex<Node>>) -> Result<Optio
     let header = node
         .lock()
         .unwrap()
-        .get_block_by_view(block)?
+        .get_block_by_number(block)?
```

Comment:
view should not be used outside of consensus.rs as we index blocks by number now